README.md

```diff
@@ -172,6 +172,7 @@ To use an external S3 compatible object store as primary storage, set the follow
 - `OBJECTSTORE_S3_PORT`: The port that the object storage server is being served over
 - `OBJECTSTORE_S3_SSL` (default: `true`): Whether or not SSL/TLS should be used to communicate with object storage server
 - `OBJECTSTORE_S3_REGION`: The region that the S3 bucket resides in.
+- `OBJECTSTORE_S3_STORAGE_CLASS` (default: `STANDARD`): The storage class to apply to all objects stored. See S3 [`--storage-class`](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html#options).
```

Comment:
Assuming the options are the same as those of the AWS CLI `aws s3 cp --storage-class`...